src/KBmag.jl

```diff
@@ -2,15 +2,18 @@ module KBmag
 
 using Libdl
 
-if isfile(joinpath(dirname(@__FILE__), "..", "deps", "deps.jl"))
-    include("../deps/deps.jl")
+deps_file = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
+
+if isfile(deps_file)
+    include(deps_file)
 else
-    error("KBmag not properly installed. Please (re) build KBmag and restart julia")
+    error("KBmag not properly installed. Please (re) build KBmag and restart Julia")
 end
 
-include("types.jl")
-include("RWS.jl")
+include("types.jl")     # struct types that reflect original structs from fsalib
+include("RWS.jl")       # loading/saving and preparing rewriting systems
+include("reduce.jl")    # word reduction
 
-include("debug.jl")
+include("debug.jl")     # independent debug functions (ready to be removed at any moment)
```

Comment:
update after renaming;

also: can you add the `BinWrapper.jl` if you have it lying around?

Reply:
BinWrapper.jl is not ready yet.